### PR TITLE
Pub/Sub code sample

### LINKED ISSUE #146: Add more examples to spring-cloud-gcp-examples
- [x] Trace
- [x] CloudSQL
- [ ] Pub/Sub
- [x] Spring Integration w/ Pub/Sub

### PR DESCRIPTION
**Pull request by joaoandremartins**
Fixes #146

### CODE REVIEW

File: spring-cloud-gcp-examples/spring-cloud-gcp-pubsub-example/src/main/java/com/example/WebController.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		this.pubSubAdmin.createTopic(topicName);
+
+		return new RedirectView("/");
+	}
+
+	@PostMapping("/createSubscription")
+	public RedirectView createSubscription(@RequestParam("topicName") String topicName,
+			@RequestParam("subscriptionName") String subscriptionName) {
+		this.pubSubAdmin.createSubscription(subscriptionName, topicName);
+		this.pubSubTemplate.subscribe(subscriptionName,
```
Comment (artembilan) [inline spring-cloud-gcp-examples/spring-cloud-gcp-pubsub-example/src/main/java/com/example/WebController.java]: You say `createSubscription`, but why you `subscribe` here as well. Unconditionally.
Thanks

File: spring-cloud-gcp-examples/spring-cloud-gcp-pubsub-example/src/main/java/com/example/PubSubApplication.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.io.IOException;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class PubSubApplication {
+
+	public static void main(String[] args) throws IOException {
```
Comment (artembilan) [inline spring-cloud-gcp-examples/spring-cloud-gcp-pubsub-example/src/main/java/com/example/PubSubApplication.java]: What an `IOException` do we have here?

Comment (joaoandremartins) [reply spring-cloud-gcp-examples/spring-cloud-gcp-pubsub-example/src/main/java/com/example/PubSubApplication.java]: Good catch!

**Comment by joaoandremartins** [comment]
@artembilan anything else needs changing?

**Pull request merged**